src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/VaccinationEntry+Extension.swift

```diff
@@ -19,6 +19,7 @@ extension VaccinationEntry {
 		"\(doseNumber) of \(totalSeriesOfDoses)"
 	}
 
+	// if administered with one dost of AstraZeneca, Moderna or BioNTech as recovery vaccination
```

Comment:
```suggestion
	// if administered with one dose of AstraZeneca, Moderna or BioNTech as recovery vaccination
```

I like the many comments :)